src/components/user/list.vue

```diff
@@ -174,7 +175,8 @@ export default {
       "Web Users have accounts on this website to oversee and administer the Projects on this server. Administrators can manage anything on the site. Users with no Sitewide Role can be given a Role on any Project, from that Project’s settings. Sitewide Administrators and some Project Roles can use a web browser to fill out Forms. To submit data through an application such as {collect}, create App Users for each Project."
     ],
     "header": {
-      "sitewideRole": "Sitewide Role"
+      "sitewideRole": "Sitewide Role",
+      "lastActive": "Last active"
```

Comment:
```suggestion
      "lastActive": "Last Active"
```

I think we usually title-case column headers.